rgd/settings.py

```diff
@@ -90,6 +90,12 @@ def before_binding(configuration: Type[ComposedConfiguration]):
 
     CELERY_WORKER_SEND_TASK_EVENTS = True
 
+    RGD_FILE_FIELD_PREFIX = values.Value(
+        default=None,
+        environ=True,
+        environ_prefix='DJANGO',
```

Comment:
FYI, these are the default values of `environ` and `environ_prefix`. They typically do not deviate from these values, so it's surprising to see them explicitly set here. They could be omitted as arguments. See: https://django-configurations.readthedocs.io/en/latest/values/#configurations.values.Value

Reply:
@banesullivan

Reply:
Thanks for pointing this out. This was unclear to me and to be safe, I mimicked what I saw right above it: https://github.com/ResonantGeoData/ResonantGeoData/blob/42e6180d82d0e4e0052bfdf2e57b0667f7028f95/rgd/settings.py#L84

I have added #375 to clean these up